batch7crud-roll9/src/main/java/com/lftechnology/batch7crud/controller/StudentControllerServlet.java

```diff
@@ -0,0 +1,179 @@
+import javax.servlet.http.HttpServletResponse;
+import javax.swing.*;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by sanjay on 1/18/16.
+ */
+@WebServlet("/students/*")
+public class StudentControllerServlet extends HttpServlet {
+    protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        String pathInfo = request.getPathInfo();
+        String[] pathParts=null;
+        if(pathInfo !=null){
+            pathParts = pathInfo.split("/");
+            System.out.println("splitting the path");
+        }
+        else{
+            System.out.println("no the path to split null");
+        }
+       if(pathParts.length == 2 && pathParts[1].equals("create")){
+            createProcess(request, response);
+       }
+       else if(pathParts.length == 3 && pathParts[2].equals("delete"))
+       {
+           if(pathParts[1].matches("[-+]?\\d*\\.?\\d+"))
+           {
+               System.out.println("center in a number");
+               deleteProcess(request,response,Integer.parseInt(pathParts[1]));
+           }
+           else{
+               System.out.println("!!!center in a not number");
+           }
+       }
+       else if(pathParts.length == 3 && pathParts[2].equals("edit"))
+       {
+           if(pathParts[1].matches("[-+]?\\d*\\.?\\d+"))
+           {
+               System.out.println("center in a number---edit");
+               editProcess(request, response, Integer.parseInt(pathParts[1]));
+           }
+           else{
+               System.out.println("!!!center in a not number");
+           }
+
+       }
+
+    }
+
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        String pathInfo = request.getPathInfo();
+        String[] pathParts=null;
+        if(pathInfo !=null){
+            pathParts = pathInfo.split("/");
+            System.out.println("splitting the path");
+        }
+        else{
+            System.out.println("no the path to split null");
+        }
+        if(pathInfo == null) {
+            if(request.getParameter("page") == null) {
+                int page = 1;
+                list(request, response, page);
+            }
+            else{
+                list(request, response, Integer.parseInt(request.getParameter("page")));
+            }
+        }
+        else if(pathParts.length == 2 && pathParts[1].equals("create")){
+            create(request,response);
+        }
+        else if(pathParts.length == 3 && pathParts[2].equals("edit"))
+        {
+            if(pathParts[1].matches("[-+]?\\d*\\.?\\d+"))
+            {
+                System.out.println("center in a number");
+                edit(request, response, Integer.parseInt(pathParts[1]));
+            }
+            else{
+                System.out.println("!!!center in a not number");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS106)